server/app/services/email/graph/GraphApiEmailClient.java

```diff
@@ -107,7 +111,18 @@ public void send(ImmutableList<String> toAddresses, String subject, String bodyT
       sendMailPostRequestBody.setMessage(message);
       // This can be a configurable value if that is something the team wants.
       sendMailPostRequestBody.setSaveToSentItems(false);
-      client.get().me().sendMail().post(sendMailPostRequestBody);
+      if (settingsManifest.getGraphApiEmailAccount().isEmpty()) {
+        logger.error(
+            "GRAPH_API_EMAIL_ACCOUNT is not set. This is needed to send emails through graph API");
+        client.get().me().sendMail().post(sendMailPostRequestBody);
```

Comment:
(question) Should this be calling `me()` still?

Reply:
Yeah - figured if an email isn't specified we can use the root account. Updated to make that clearer.